core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -342,7 +342,7 @@ package object config {
         "a property key or value, the value is redacted from the environment UI and various logs " +
         "like YARN and event logs.")
       .regexConf
-      .createWithDefault("(?i)secret|password|url|user|username".r)
+      .createWithDefault("(?i)secret|password".r)
```

Comment:
If the users want to see the values, they can change the default by themselves, right?

Reply:
Not sure what you're trying to imply? The default is redacting too much. That needs to be fixed.

Reply:
Normally, making it stricter is not a big deal. However, if we want to relax it, this could be a big deal since users might expect these variables have been hided by default. cc the original reviewers and author @onursatici @ash211 @hvanhovell @jiangxb1987

Reply:
URIs could contain the access keys and usernames could have potentially personal identifiable information. Could we do not change the default?